chcemvediet/templates/main/homepage/about.html

```diff
@@ -41,4 +41,8 @@
       {% include "main/homepage/login.html" %}
     </div>
   {% endif %}
+  <div class="col-md-8 col-sm-6">
+    <h2 id="vybrane-pripady" class="text-center">Vybrané prípady</h2>
+    {% include "main/homepage/case-studies/items.html" %}
```

Comment:
Nadpisy zvykneme mat vo vnutri pomocneho templatu. Vid napriklad `main/homepage/funding.html`. Sprav template `main/homepage/case-studies/main.html`, kde bude nadpis aj items.

Edit: Pozeram, ze blogs a funding je tiez spraveny tak, ze nadpis je mimo pomocneho templatu. To nie je dobre. Lepsie je ked nadpis je na jednom mieste s templatom. Pls v separatnom PR to pretob aj na blogs a funding.

Reply:
@martinmacko47 Editovat aj `fundings`? Tam je nadpis v pomocnom template.
A keby som len premenoval subor `main/homepage/case-studies/items.html` na `main.html`, asi zbytocne vytvarat dalsi subor.

Reply:
Ano, kludne subor premenuj na `main.html`.

Co treba editovat vo fundings, ked hovoris, ze tam je nadpis dnu?

Reply:
Asi si mal na mysli **didyouknow** a nie **funding** v komentari casti "Edit:".
A s tym premenovanim suboru beriem spät, nechal by som `casestudies/items.html`

Reply:
@viliambalaz Ano, mal som na mysli "didyouknow". Teda tie, kde su nadpisy vysunute do vonkajsieho templatu. Uprav to v seperatnom PR pls.